expensereport-swift/Tests/ExpenseReportTests/ExpenseReportTests.swift

```diff
@@ -1,7 +1,7 @@
 @testable import ExpenseReport
 import XCTest
 
-class ExpenseReportTests: XCTestCase {
-    func testPrintReport() {
+final class ExpenseReportTests: XCTestCase {
+    func testPrintReport() throws {
```

Comment:
XCTest is adding more helpers which throw, so I've shifted to declaring all tests as `throws` so I can use them without modifying the test declarations.